realtime/src/app/errors.ts

```diff
@@ -1,11 +1,11 @@
 import * as DomainErrors from "@domain/errors"
 import * as ExchangesErrors from "@domain/exchanges/errors"
-import * as ProvidersErrors from "@domain/providers/errors"
 import * as PrimitivesErrors from "@domain/primitives/errors"
+import * as PriceErrors from "@domain/price/errors"
 
 export const ApplicationErrors = {
```

Comment:
Are we adding an error-map switch to do any sort of mapping in this repo as well?